packages/opencensus-exporter-stackdriver/README.md

```diff
@@ -55,6 +56,58 @@ or
 tracing.registerExporter(exporter).start();
 ```
 
+Viewing your traces:
+
+With the above you should now be able to navigate to the Stackdriver UI at: <https://console.cloud.google.com/traces/traces>
+
+# OpenCensus Stackdriver Stats/Metrics Exporter
+## Installation
+
+Install OpenCensus Stackdriver Exporter with:
+```bash
+npm install @opencensus/core
+npm install @opencensus/exporter-stackdriver
+```
+
+## Usage
+
+To use Stackdriver as your exporter, make sure you have enabled [Stackdriver Monitoring](https://cloud.google.com/monitoring/docs/quickstart) on Google Cloud Platform. Enable your [Application Default Credentials](https://cloud.google.com/docs/authentication/getting-started) for authentication with:
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS=path/to/your/credential.json
+```
+
+Instance the exporter on your application.
```

Comment:
Nit: did you mean "Install the exporter ..."?

Reply:
Copied from top example (Trace Exporter), amended both the sentence.

Reply:
Yes, I did.